src/Hosting/Hosting/src/Internal/HostingMetrics.cs

```diff
@@ -197,4 +196,60 @@ private static object GetBoxedStatusCode(int statusCode)
 
         return statusCode;
     }
+
+    private static readonly FrozenDictionary<string, string> KnownMethods = FrozenDictionary.ToFrozenDictionary(new[]
+    {
+        KeyValuePair.Create(HttpMethods.Connect, HttpMethods.Connect),
```

Comment:
otel  requires providing a way to override the list 

> If the HTTP instrumentation could end up converting valid HTTP request methods to `_OTHER`, then it MUST provide a way to override
the list of known HTTP methods. If this override is done via environment variable, then the environment variable MUST be named
OTEL_INSTRUMENTATION_HTTP_KNOWN_METHODS and support a comma-separated list of case-sensitive known HTTP methods
(this list MUST be a full override of the default known method, it is not a list of known methods in addition to the defaults).

https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md?plain=1#L96-L199

LMK your thoughts on this. At least I should update the description in my PR.

Reply:
As I mentioned in the email, we don't have time to add configuration for .NET 8. I'm happy reporting other values as "_OTHER" and then configuration can be added in the future if people ask for it.

Reply:
Might be able to add this configuration alongside config for enabling capturing host header info